### DISCUSSION TITLE
UDFs in others languages like postgres do with PLV8

### DISCUSSION THREAD

**Discussion by calebeaires** [geldata/gel#3253] [Ideas]
edgedb is awasome. After read its documentation I understand that it is not the propose but: is there some way to add support to user defined functions (UDFs) made in other languages, like Javascript

**Comment by elprans** [comment]
EdgeDB v1.0 does not support non-EdgeQL functions.  Future versions will likely introduce an extension mechanism which would allow something like this.

**Reply by calebeaires** [reply]
Thanks. I've been playing with edgedb. Great lib.

**Reply by calebeaires** [reply]
hey @elprans, taking with friends about this edgedb, we turn back on this subject. Is this feature included at the roadmap already?

**Reply by calebeaires** [reply]
@elprans,

Hope this feature is on your radar!
